src/pages/PerformanceDetails/sections/EarningSection/index.tsx

```diff
@@ -0,0 +1,333 @@
+        correspondingPastSwapFee.tokens.times(rates[swapFee.info.symbol].toNumber()),
+      );
+    }
+
+    return true;
+  });
+
+  let expensesTotalCurrent = new BigNumber(0);
+  let expensesTotalPast = new BigNumber(0);
+
+  earningsCurrentPeriod?.expenses.map(expense => {
+    const correspondingPastExpense = earningsPastPeriod?.expenses.filter(
+      pastExpense => pastExpense.info.symbol === expense.info.symbol,
+    )[0];
+
+    if (rates) {
+      expensesTotalCurrent = expensesTotalCurrent.plus(expense.tokens.times(rates[expense.info.symbol].toNumber()));
+    }
+    if (rates && correspondingPastExpense) {
+      expensesTotalPast = expensesTotalPast.plus(
+        correspondingPastExpense.tokens.times(rates[expense.info.symbol].toNumber()),
+      );
+    }
+
+    return true;
+  });
+
+  return (
+    <BoxPanel bg="bg2" mt={10} mb={10}>
+      <Flex alignItems={'center'} justifyContent={'space-between'}>
+        <Typography variant="h2">Earnings</Typography>
+        <Text>
+          {`Income earned `}
+          <ClickAwayListener onClickAway={closeDropdown}>
+            <UnderlineTextWithArrow
+              onClick={handleToggle}
+              text={`past ${timePeriod.displayText}`}
+              arrowRef={arrowRef}
+            />
+          </ClickAwayListener>
+          <DropdownPopper show={Boolean(anchor)} anchorEl={anchor} placement="bottom-end">
+            <MenuList>
+              {Object.keys(earningPeriods).map(
+                period =>
+                  earningPeriods[period].days !== timePeriod.days && (
+                    <MenuItem key={period} onClick={() => handlePeriodChange(period)}>
+                      {earningPeriods[period].displayText}
+                    </MenuItem>
+                  ),
+              )}
+            </MenuList>
+          </DropdownPopper>
+        </Text>
+      </Flex>
+      <ScrollHelper>
+        <IncomeGrid>
+          <GridItemHeader>INCOME</GridItemHeader>
+          <GridItemHeader>
+            {formattedDates.current.dateStart} - {formattedDates.current.dateEnd}
+          </GridItemHeader>
+          <GridItemHeader>
+            {formattedDates.past.dateStart} - {formattedDates.past.dateEnd}
+          </GridItemHeader>
+        </IncomeGrid>
+
+        <IncomeGrid>
+          <GridItemStrong>Loan fees</GridItemStrong>
+          <GridItemStrong>
+            {earningsCurrentPeriod ? (
+              displayValueOrLoader(loanFees, 1)
+            ) : (
+              <StyledSkeleton animation="wave" width={100} />
+            )}
+          </GridItemStrong>
+          <GridItemStrong>
+            {earningsPastPeriod ? (
+              displayValueOrLoader(loanFeesPast, 1)
+            ) : (
+              <StyledSkeleton animation="wave" width={100} />
+            )}
+          </GridItemStrong>
+        </IncomeGrid>
+
+        <IncomeGrid>
+          <GridItemLight>Balanced Dollars (bnUSD)</GridItemLight>
```

Comment:
```suggestion
          <GridItemLight>Balanced Dollar (bnUSD)</GridItemLight>
```

I always prefer to keep the `s` on Balanced Dollars, Balance Tokens, and Omm Tokens, but let's remove it here to keep it consistent with the rest of the Stats page.

Reply:
fixed